<launch>
    <!--Parameters-->
    <arg name="frame_id" default="map" />
    
    <!--node launch-->
    <node pkg= "decawave" ns="uwb" name="dataFiltered" type="filter_UWBpoint_SX10" output="screen">
        <param name="frame_id" value="$(arg frame_id)" />
        <param name="filterInput1" value="/uwb/point0/decaPoint" />
        <param name="filterInput2" value="/trimble/SX10/data" />

    </node>

</launch>
